### PR TITLE
Clesaec/tdi 45389 clean pom

### PR DESCRIPTION
**Pull request by clesaec**
https://jira.talendforge.org/browse/TDI-45389

First step to clean pom on connectors-se module

### CODE REVIEW

File: dynamics-crm/pom.xml [hunk 1]
```diff
@@ -58,89 +58,6 @@
...
-                        </configuration>
-                    </execution>
-
-                    <execution>
-                        <id>pre-integration-test</id>
-                        <goals>
-                            <goal>prepare-agent-integration</goal>
-                        </goals>
-                        <configuration>
-                            <destFile>${project.build.directory}/coverage-reports/jacoco-it.exec</destFile>
```
Comment (ypiel-talend) [inline dynamics-crm/pom.xml:83]: Maybe we should add it in parent pom ?

**Review by ypiel-talend — COMMENTED** [review]
Some remarks

File: pom.xml [hunk 2]
```diff
@@ -1148,5 +1148,43 @@
...
+      <id>failsafe</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <version>${surefire.version}</version>
+            <configuration>
+              <includes>
+                <include>**/*IT.java</include>
```
[the remark this answers is not in the document]
Comment (clesaec) [reply pom.xml]: ok, removed

**Review by rlecomte-talend — APPROVED** [review]
LGTM

File: dynamics-crm/pom.xml (hunk 1 above)
Comment (clesaec) [reply dynamics-crm/pom.xml:83]: added

**Review by rlecomte-talend — APPROVED** [review]
LGTM

**Pull request merged**